src/display/editor/editor.js

```diff
@@ -881,6 +869,26 @@ class AnnotationEditor {
       const id = (tooltip.id = `alt-text-tooltip-${this.id}`);
       button.append(tooltip);
       button.setAttribute("aria-describedby", id);
+
+      const DELAY_TO_SHOW_TOOLTIP = 500;
+      button.addEventListener("mouseenter", () => {
+        this.#altTextTooltipTimeout = setTimeout(() => {
+          this.#altTextTooltipTimeout = null;
+          this.#altTextTooltip.classList.add("show");
+          this._uiManager.reportTelemetry({
+            type: "editing",
+            subType: "altText",
```

Comment:
It seems that the casing is inconsistent throughout the patch, with both `subtype` and `subType` being used, which is probably not intended.

Please just use the *all lowercase* version everywhere, since it seems (to me) that it should reduce the risk of future bugs.
```suggestion
            subtype: "altText",
```